res/xml/method.xml

```diff
@@ -4,7 +4,7 @@
   <subtype android:label="%s" android:languageTag="be" android:imeSubtypeLocale="be_BY" android:imeSubtypeMode="keyboard" android:isAsciiCapable="true" android:imeSubtypeExtraValue="default_layout=ru_jcuken,extra_keys=ґ|є|і|ї|ў"/>
   <subtype android:label="%s" android:languageTag="bg" android:imeSubtypeLocale="bg_BG" android:imeSubtypeMode="keyboard" android:isAsciiCapable="true" android:imeSubtypeExtraValue="default_layout=bgph1,extra_keys=€"/>
   <subtype android:label="%s" android:languageTag="bn" android:imeSubtypeLocale="bn_BD" android:imeSubtypeMode="keyboard" android:isAsciiCapable="true" android:imeSubtypeExtraValue="default_layout=qwerty,extra_keys=৳"/>
-  <subtype android:label="%s" android:languageTag="cs" android:imeSubtypeLocale="cs_CZ" android:imeSubtypeMode="keyboard" android:isAsciiCapable="true" android:imeSubtypeExtraValue="default_layout=qwertz_cs,extra_keys=a|accent_cedille|accent_circonflexe"/>
+  <subtype android:label="%s" android:languageTag="cs" android:imeSubtypeLocale="cs_CZ" android:imeSubtypeMode="keyboard" android:isAsciiCapable="true" android:imeSubtypeExtraValue="default_layout=qwertz_cs_minimal"/>
```

Comment:
In the issue you mention the Slovak language. Could you add it as a subtype ?

Reply:
I have no problem with creating a minimal slovak keyboard, but as I am not a slovak speaker, I can not test by using it for a month (as I did with the czech layout).

The only problem I have is, that I don't know what a subtype is.

Reply:
No need to make a Slovak layout! I'm thinking of using qwerty or qwertz with the appropriate `extra_keys`.
I asked this just in case you knew, don't worry about that :)

A subtype is a line of `method.xml` similar to the one you modified. It specifies the preferred layout for a language and the keys that should be added to other layouts automatically.

Reply:
Oh, ok, I already looked into it a little bit, and it seems that [slovak](https://en.wikipedia.org/wiki/File:KB_Slovak.svg) and [czech](https://cs.wikipedia.org/wiki/Soubor:Windows_Czech_keyboard_layout.svg) layouts are different to such extend, that slovak would deserve its own layout.

Reply:
Thanks!